### ISSUE TITLE
Exception Occurred Version ^5.0

### ISSUE THREAD

**Issue by ssrgspdkt** [pgrimaud/instagram-user-feed#58]
Last 2 days it was working properly. 
Currently Showing as follows/pgrimaud/instagram-user-feed/src/Instagram/Hydrator/HtmlHydrator.php:33 Invalid argument supplied for foreach() (2)

**Comment by christiaen-francois** [comment]
same to me :( I've upgrade to the new version and it says that my IP is blocked

**Comment by pgrimaud** [comment]
Damn :/ 

Which version did you use before upgrade?
Are you using cache manger?
Are you using login process?

**Comment by christiaen-francois** [comment]
the version 5.0 just before your last update.
Yes the cache manager is used.
No login process used.

But 24h later, it seems to work fine ! I didn't make any changes.

**Comment by christiaen-francois** [comment]
OMG no ....

Fatal error: Uncaught Instagram\Exception\InstagramAuthException: Instagram blocked your IP. .... /vendor/pgrimaud/instagram-user-feed/src/Instagram/Transport/HtmlTransportFeed.php on line 73

**Comment by pgrimaud** [comment]
In the last version, I catch this error which wasn't catch in the previous one.

Which hosting are you using? Dedicated ?

**Comment by christiaen-francois** [comment]
I'm on a shared hosting. The website is www.coterue.be and is under development. I don't have any visitors yet.
Some time all is right and other time the feed is empty 
<img width="353" alt="Capture d’écran 2020-03-31 à 17 47 51" src="https://user-images.githubusercontent.com/59294881/78046823-eba54200-7377-11ea-8ea9-284bf26df40f.png">

<img width="670" alt="Capture d’écran 2020-03-31 à 17 51 01" src="https://user-images.githubusercontent.com/59294881/78047083-3757eb80-7378-11ea-8e7f-ee7b3222502c.png">

**Comment by pgrimaud** [comment]
Ok, Some people may have the same issue with shared hosting. https://github.com/pgrimaud/instagram-user-feed/issues/45 https://github.com/pgrimaud/instagram-user-feed/issues/46 https://github.com/pgrimaud/instagram-user-feed/issues/47

Is it working on your local environment? 
 
If yes, could you try to create a dummy Instagram account, verify the associated email and try the login method ?

I just updated the README.md with a login process example  : 
https://github.com/pgrimaud/instagram-user-feed#usage-of-version-50

```php
$api = new Instagram\Api();
$api->login('username', 'password'); // optional, may be required on shared hosting
$api->setUserName('coterueheusy');

$feed = $api->getFeed();

print_r($feed);
```

**Comment by christiaen-francois** [comment]
I receive the connexion notification email and receive this fallback : 

`Client error: `POST https://www.instagram.com/accounts/login/ajax/` resulted in a `400 Bad Request` response: {"message": "checkpoint_required", "checkpoint_url": "/challenge/17921776/qXZf26UOLN/", "lock": false, "status": "fail"} `

**Comment by pareshsojitra** [comment]
I'm getting error with login process option and I'm using it with cache manager.

**Site 1 :**
Client error: `POST https://www.instagram.com/accounts/login/ajax/` resulted in a `403 Forbidden` response: {"message": "Please wait a few minutes before you try again.", "status": "fail"} 

**Site 2:**
Wrong login / password

**Comment by adambichler** [comment]
> I'm getting error with login process option and I'm using it with cache manager.
>
> Client error: `POST https://www.instagram.com/accounts/login/ajax/` resulted in a `403 Forbidden` response: {"message": "Please wait a few minutes before you try again.", "status": "fail"}

I'm gettting this error too:
Uncaught GuzzleHttp\Exception\ClientException: Client error: `POST https://www.instagram.com/accounts/login/ajax/` resulted in a `403 Forbidden` response: {"message": "Please wait a few minutes before you try again.", "status": "fail"} in "PATH TO MY GUZZLE FOLDER HERE"

Caching is enabled and I'm using the login method.

**Comment by pgrimaud** [comment]
Hello, I released a new version (6.0). Could you upgrade to this one?

**Comment by pwnz22** [comment]
```Undefined property: stdClass::$authenticated```

I'm getting this error while using this package on the server.

```$cachePool = new FilesystemAdapter('Instagram', 0, __DIR__ . '/../
$api = new Api($cachePool);
$api->login(login, pass);```

Any suggestions?
